### DISCUSSION TITLE
Update the link of django-rest-auth in the documentation website

### DISCUSSION THREAD

**Discussion by jefcolbi** [encode/django-rest-framework#8141] [Ideas & Suggestions]
On the this page [https://www.django-rest-framework.org/community/third-party-packages/](https://www.django-rest-framework.org/community/third-party-packages/) of the documentation, you are referencing [https://github.com/Tivix/django-rest-auth/](https://github.com/Tivix/django-rest-auth/) as a package. But this package is now unmaintained.

I would be good if you update it with the url of the fork. See this issue for more infos [https://github.com/Tivix/django-rest-auth/issues/568](https://github.com/Tivix/django-rest-auth/issues/568)

Thanks 🙏

**Comment by lovelydinosaur** [comment]
Happy to accept pull requests updating the docs there, sure.

**Comment by jefcolbi** [comment]
Alright
